packages/client/components/TaskFooterIntegrateMenuList.tsx

```diff
@@ -70,13 +70,13 @@ const Label = styled('div')({
 const getValue = (
   item: NonNullable<TaskFooterIntegrateMenuList_suggestedIntegrations['items']>[0]
 ) => {
-  const jiraItemName = item?.remoteProject?.name ?? ''
+  const jiraItemName = item?.projectKey ?? ''
   const githubName = item?.nameWithOwner ?? ''
-  const name = jiraItemName ?? githubName
+  const name = jiraItemName || githubName
```

Comment:
The nullish coalescing operator `??` only returns the right-hand side of the operand when the left-hand side is `null` or `undefined`. So, when `jiraItemName` was `""`, we were still returning the `jiraItemName`. With the logical or `||` operator, we return the `githubName` when `jiraItemName` is falsy, e.g. when it's an empty string.